src/Storage.js

```diff
@@ -135,7 +131,7 @@ const startCassandraStorage = async (contactPoints, localDataCenter, keyspace) =
         localDataCenter,
         keyspace,
     })
-    await callbackToPromise(cassandraClient.connect.bind(cassandraClient))
+    await cassandraClient.connect.bind(cassandraClient)
```

Comment:
I wonder what's happening here? Is this intended to be just `await cassandraClient.connect()`? I would assume that all the `bind` hacks were due to `callbackToPromise`.